# 舵轮模型相关参数

k_red_ratio:    40 # 行走电机减速比

radius_steering_wheel: 0.11 # 主舵轮的半径

L_base:   1.3 # 单位:米,轴距 主动轮到车体中心的距离
L_base_y:   0 # 单位:米,主动轮在y轴上的偏距，位于车体左边为正数

L_passive_tread:  1.42   # 主动轮之间轴距离
L_passive_base:   0.7   # 从动轮到base之间的距离

L_vehicle_length: 2.4   #车体长度
L_steering_front: 0.29  #前轮到车头的距离 

max_wheel_speed: 1.0
max_steer_angle: 3.14